app/exporters/disaggregated_dhis2_exporter.rb

```diff
@@ -1,11 +1,33 @@
 class DisaggregatedDhis2Exporter
+  BUCKETS = (15..75).step(5)
+
   def self.export
     exporter = Dhis2Exporter.new(
       facility_identifiers: FacilityBusinessIdentifier.dhis2_org_unit_id,
       periods: (current_month_period.advance(months: -24)..current_month_period),
-      data_elements_map: CountryConfig.current.fetch(:dhis2_data_elements)
+      data_elements_map: CountryConfig.current.fetch(:disaggregated_dhis2_data_elements)
     )
     exporter.export do |facility_identifier, period|
+      region = facility_identifier.facility.region
+      {
+        cumulative_assigned: disaggregated_counts(PatientStates::CumulativeAssignedPatientsQuery.new(region, period)),
+        controlled: disaggregated_counts(PatientStates::ControlledPatientsQuery.new(region, period)),
+        uncontrolled: disaggregated_counts(PatientStates::UncontrolledPatientsQuery.new(region, period)),
+        missed_visits: disaggregated_counts(PatientStates::MissedVisitsQuery.new(region, period))
```

Comment:
If these new query classes are only used for this disaggregation, it seems like a lot of moving pieces for this one purpose. Have we considered keeping everything in one place for now, and extracting separate query classes if we start re-using the queries in other places? (eg. other DHIS2 pipes, the "repository" itself, etc).

I'm thinking something like:

```ruby

# In the DHIS2 Exporter
def self.query
  @query ||= PatientStates::DisaggregatedPatientCountQuery.new(
    region: region,
    period: period,
    age_buckets: BUCKETS
  )
end

def self.export
  # ...
  cumulative_assigned: query.cumulative_assigned,
  controlled: query.controlled,
  uncontrolled: query.uncontrolled,
  # ...
end

# And the query class would look like
class PatientStates::DisaggregatedPatientCountQuery
  def initialize(region:, period:, age_buckets:)
     # initialize stuff
  end

  def cumulative_assigned
    # the cumulative assigned query
  end

  def controlled
    cumulative_assigned.where(criteria_for_controlled_patients)
  end
end
```

Sure, it's weird for a "query" to have multiple queries inside, but I'd rather change the name of the class and still keep everything in one place, than split it up just yet. What do you think?